### PR TITLE
Complete post section

### PR DESCRIPTION
**Pull request by Ashnaali3255**
- Add Images for posts.
- Add titles for posts.
- Add tags for posts.
- Add buttons for posts.
- Add button for resume.
- Add an about us section.
- Add languages.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -92,28 +88,194 @@
...
+    border: 1px solid #396df2;
+    border-radius: 16px;
+    font-size: 17px;
+    color: #396df2;
+  }
+
+  .about {
+    margin: 0 24px;
+    color: #172b4d;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css:202]: - [ ] Kindly give your about section a `border` and `border-top-right-radius` as seen in your chosen Figma design 🙏.

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 2]
```diff
@@ -92,28 +88,194 @@
...
+  .arrow {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #000;
+    justify-content: space-between;
```
Comment (Ugboaja-Uchechi) [inline style.css:252]: - [ ] Kindly align your languages section as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Ashnaali3255, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -20,20 +20,121 @@
...
     <main>
  <section class="main-section">
-  <h2 class="name">I'm Ashna Ali <br>
-   Glad to see you!</h2>
+  <h1 class="name">I'm Ashna Ali <br>
+   Glad to see you!</h1>
   <p class="description">
     I'm a software developer! I can help you build a product , feature or website Look through some of my work and
    experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
   <h3>LET'S CONNECT</h3>
```
Comment (Alfredbis29) [inline index.html:28]: - [ ] kudos for making some changes required in the previous review, but you still need to improve on something but you are almost there, As required for this project, your section uses the font and colors (background and font) defined for the template . kindly fix this for your project to match the Figma design :wink:  don't forget to take care of the background images as displayed on the figma design below :wink: 


Your Design 
color

Figma design
figmaheader

File: index.html [hunk 4]
```diff
@@ -20,20 +20,121 @@
...
+     <li><img src="./Counter.jpg" alt="counter">Back End Dev</li>
+     <li><img src="./Counter.jpg" alt="counter">2022</li>
+    </ul>
+    <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     <ul class="list-flex">
+      <li class="language">html</li>
+      <li class="language">css</li>
+      <li class="language">javascript</li>
+     </ul>
+      <button>See project</button>
```
Comment (Alfredbis29) [inline index.html:53]: - [ OPTIONAL] As required in the previous review, Kindly make sure your button looks as close as possible :wink: 
Your design
butti
Figma 
figmabutti

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @Ashnaali3255 , 

Good job so far, I can see that you put in a lot of energy into this :+1: 
However, there are some few issues you need to work on in order to proceed to the next project, but believe me, you are almost there. 

 soclose 

## Required Changes :recycle: 
Check under the review

## Optional Suggestions
_Check under the review

Cheers and Happy coding :clap: :clap: :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @alfredbis29  in your question so I can receive the notification.


#### Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

#
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 5]
```diff
@@ -15,25 +15,126 @@
...
    experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-  <h3>LET'S CONNECT</h3>
+  <h3 class="connect">LET'S CONNECT</h3>
   <ul class="links">
+   <li><a title="twitter" href="https://twitter.com/Ashna_Ali1"><img src="./Twitter.jpg" alt="twitter"></a></li>
+   <li><a title="linkedIn" href="https://www.linkedin.com/in/ashna-ali-342151255/"><img src="./linkedin.jpeg" alt="linkedin"></a></li>
+   <li><a title="Medium" href="https://medium.com/Ashna_Ali1"><img src="./Medium.jpg" alt="medium"></a></li>
+   <li><a title="github" href="https://github.com/Ashnaali3255/Portfolio/tree/Mobile-Branch"><img src="./Github.jpg" alt="github"></a></li>
+   <li><a title="Angelist" href="https://angelist.com/Ashna_Ali1"><img src="./Angel.jpg" alt="angelist"></a></li>
+  </ul>
```
Comment (mckent05) [inline index.html:35]: - Just as the previous reviewers suggested, please make sure the background images for this section are aligned as it is in the Figma design 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
bvsbvhbsbv

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Ashnaali3255,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost there



 ## Highlights
- All linter checks are passing ✔️ 
- You have used the right workflow for this milestone 👏 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Ashnaali3255 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**